src/subdataframe/subdataframe.jl

```diff
@@ -136,13 +136,16 @@ end
 
 Base.copy(sdf::SubDataFrame) = parent(sdf)[rows(sdf), parentcols(index(sdf), :)]
 
-function without(df::SubDataFrame, icols::Vector{<:Integer})
-    newcols = setdiff(1:ncol(df), icols)
-    view(df, newcols)
-end
 deleterows!(df::SubDataFrame, ind) =
     throw(ArgumentError("SubDataFrame does not support deleting rows"))
 
-DataFrame(sdf::SubDataFrame) = sdf[:, :]
+function DataFrame(sdf::SubDataFrame, copycolumns::Bool=true)
+    if copycolumns
+        sdf[:, :]
+    else
+        DataFrame(eachcol(sdf, false), names(sdf), copycolumns=false)
+    end
+end
 
-Base.convert(::Type{DataFrame}, sdf::SubDataFrame) = DataFrame(sdf)
+Base.convert(::Type{DataFrame}, sdf::SubDataFrame, copycolumns::Bool=true) =
```

Comment:
Same question here as above

Reply:
@nalimilan - actually this is not the same if we decided to remove `copycolumns` in the other `convert`, because if we would not copy (the default in Base) we would create a `DataFrame` of views. So do we set `copycolumns=true` here as the only option?

Reply:
OK - I will remove this.